docs/diagrams/ModelClassDiagram.puml

```diff
@@ -4,49 +4,18 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
```

Comment:
nit: This should be renamed to QuestionClassDiagram as its more accurate

Reply:
Ah I apologise. I accidentally edited the ModelClassDiagram T.T. I have reverted it to its original state. I have another puml file called QuestionClassDiagram